### PR TITLE
Repeatable annotations

### PR DESCRIPTION
**Pull request by RobWin**

### CODE REVIEW

File: resilience4j-annotations/src/main/java/io/github/resilience4j/ratelimiter/annotation/RateLimiter.java [hunk 1]
```diff
@@ -38,7 +39,7 @@
 
     /**
      * fallbackMethod method name.
-     *
+     *T
```
Comment (dlsrb6342) [inline resilience4j-annotations/src/main/java/io/github/resilience4j/ratelimiter/annotation/RateLimiter.java]: Please delete 😄

File: resilience4j-spring-boot-common/src/main/java/io/github/resilience4j/circuitbreaker/autoconfigure/AbstractCircuitBreakerConfigurationOnMissingBean.java [hunk 2]
```diff
@@ -81,24 +81,30 @@ public CircuitBreakerRegistry circuitBreakerRegistry(
     @ConditionalOnMissingBean
     @Conditional(value = {AspectJOnClasspathCondition.class})
     public CircuitBreakerAspect circuitBreakerAspect(CircuitBreakerRegistry circuitBreakerRegistry,
-        @Autowired(required = false) List<CircuitBreakerAspectExt> circuitBreakerAspectExtList,
+        @Autowired(required = false) List<CircuitBreakerDecoratorExt> circuitBreakerAspectExtList,
```
Comment (dlsrb6342) [inline resilience4j-spring-boot-common/src/main/java/io/github/resilience4j/circuitbreaker/autoconfigure/AbstractCircuitBreakerConfigurationOnMissingBean.java]: Could you change variable name? 
```suggestion
        @Autowired(required = false) List<CircuitBreakerDecoratorExt> circuitBreakerDecoratorExtList,
```

File: resilience4j-spring-boot-common/src/main/java/io/github/resilience4j/ratelimiter/autoconfigure/AbstractRateLimiterConfigurationOnMissingBean.java [hunk 3]
```diff
@@ -80,7 +80,7 @@ public RateLimiterRegistry rateLimiterRegistry(
     public RateLimiterAspect rateLimiterAspect(
         RateLimiterConfigurationProperties rateLimiterProperties,
         RateLimiterRegistry rateLimiterRegistry,
-        @Autowired(required = false) List<RateLimiterAspectExt> rateLimiterAspectExtList,
+        @Autowired(required = false) List<RateLimiterDecoratorExt> rateLimiterAspectExtList,
```
Comment (dlsrb6342) [inline resilience4j-spring-boot-common/src/main/java/io/github/resilience4j/ratelimiter/autoconfigure/AbstractRateLimiterConfigurationOnMissingBean.java]: Could you change variable name? 
```suggestion
        @Autowired(required = false) List<RateLimiterDecoratorExt> rateLimiterDecoratorExtList,
```

File: resilience4j-spring-boot-common/src/main/java/io/github/resilience4j/retry/autoconfigure/AbstractRetryConfigurationOnMissingBean.java [hunk 4]
```diff
@@ -91,7 +91,7 @@ public RetryRegistry retryRegistry(RetryConfigurationProperties retryConfigurati
     @ConditionalOnMissingBean
     public RetryAspect retryAspect(RetryConfigurationProperties retryConfigurationProperties,
         RetryRegistry retryRegistry,
-        @Autowired(required = false) List<RetryAspectExt> retryAspectExtList,
+        @Autowired(required = false) List<RetryDecoratorExt> retryAspectExtList,
```
Comment (dlsrb6342) [inline resilience4j-spring-boot-common/src/main/java/io/github/resilience4j/retry/autoconfigure/AbstractRetryConfigurationOnMissingBean.java]: Could you change variable name? 
```suggestion
        @Autowired(required = false) List<RetryDecoratorExt> retryDecoratorExtList,
```

File: resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerAspect.java [hunk 5]
```diff
@@ -68,124 +60,63 @@
...
     public CircuitBreakerAspect(CircuitBreakerConfigurationProperties circuitBreakerProperties,
         CircuitBreakerRegistry circuitBreakerRegistry,
-        @Autowired(required = false) List<CircuitBreakerAspectExt> circuitBreakerAspectExtList,
+        @Autowired(required = false) List<CircuitBreakerDecoratorExt> circuitBreakerDecoratorExtList,
         FallbackDecorators fallbackDecorators) {
         this.circuitBreakerProperties = circuitBreakerProperties;
-        this.circuitBreakerRegistry = circuitBreakerRegistry;
-        this.circuitBreakerAspectExtList = circuitBreakerAspectExtList;
-        this.fallbackDecorators = fallbackDecorators;
+        this.circuitBreakerDecorator = new
```
Comment (dlsrb6342) [inline resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerAspect.java]: How about making this object as bean?

File: resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/ReactorDecoratorExt.java [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/**
+ * the Reactor breaker logic support for the spring AOP Conditional on Reactor class existence on
+ * spring class loader
+ */
+public class ReactorDecoratorExt implements CircuitBreakerDecoratorExt {
```
Comment (dlsrb6342) [inline resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/ReactorDecoratorExt.java]: Why did you remove `CircuitBreaker` prefix? I think this should be `ReactorCircuitBreakerDecoratorExt`

File: resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerAspect.java (hunk 5 above)
Comment (RobWin) [reply resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerAspect.java]: No need, or? Nobody else needs this class.

File: resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/ReactorDecoratorExt.java (hunk 6 above)
Comment (RobWin) [reply resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/ReactorDecoratorExt.java]: I can revert it

File: resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerAspect.java (hunk 5 above)
Comment (dlsrb6342) [reply resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerAspect.java]: Oh then no needs.
